lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -316,4 +339,55 @@ sub _processMSIE {
 
 }
 
+# List of SQL Instances
+sub _getSqlInstancesList {
```

Comment:
This function returns a scalar, why is it called 'get a list of something' ?

Reply:
Rename it to _getSqlEdition